client/src/components/HeightClipper/HeightClipper.tsx

```diff
@@ -37,7 +40,24 @@ export class HeightClipper extends React.Component<
     };
     this.main = React.createRef();
     this.content = React.createRef();
+
+    this.debounced_mutation_callback = _.debounce(() => {
```

Comment:
Why did you need to setup mutation observer for debouncing componentDidUpdate?

And also, is this an equivalent of `_.debounce(this.componentDidUpdate)`

Reply:
Sorry forgot to answer this... The mutation observer is needed since tagging "untabbable" elements wasn't always happening because of timing issues. Now with the mutation observer, componentDidUpdate will look through the elements again to tag the missing hidden elements.